FlexMeasures/flexmeasures
# db: improve belief search with new index

## Description

Loading beliefs takes too long. Next to [a subquery improvement in timely-beliefs](https://github.com/SeitaBV/timely-beliefs/pull/166), we can also add an index on the 4 fields which usually are part of the queries. [PR TB#167](https://github.com/SeitaBV/timely-beliefs/pull/166) makes sure a db migration catches the new index.

This PR includes this migration for FlexMeasures.

## Look & Feel

`flexmeasures db upgrade`

## How to test

In [this PR](https://github.com/SeitaBV/timely-beliefs/pull/166) I describe a small script to test the speedup (between before and after this migration).

I actually did not see a giant speedup, when the subquery optimization from TB#166 is already there. Without it, it can do a 50% improvement.
